src/System.IO.IsolatedStorage/ref/System.IO.IsolatedStorage.cs

```diff
@@ -152,20 +149,6 @@ public enum IsolatedStorageScope
         Roaming = 8,
         User = 1,
     }
-
-    public enum IsolatedStorageSecurityOptions
```

Comment:
Were these in the implementation? If so you can remove them there as well. If they weren't then I would expect a baseline file update.